model/ddl.go

```diff
@@ -61,10 +61,15 @@ const (
 	ActionRepairTable                   ActionType = 29
 	ActionSetTiFlashReplica             ActionType = 30
 	ActionUpdateTiFlashReplicaStatus    ActionType = 31
+	ActionAddPrimaryKey                 ActionType = 32
+	ActionDropPrimaryKey                ActionType = 33
 )
 
-// AddIndexStr is a string related to the operation of "add index".
-const AddIndexStr = "add index"
+const (
+	// AddIndexStr is a string related to the operation of "add index".
+	AddIndexStr      = "add index"
+	AddPrimaryKeyStr = "add primary key"
+)
```

Comment:
Is it necessary to add another layer like `xxxStr`?

Reply:
I feel that there is no need at present, this two are needed in other packages.